api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimMessageSender.java

```diff
@@ -24,8 +28,8 @@
 
     private final MessageQueueClient messageQueueClient;
 
-    public void sendReportClaimMessage(Claim claim) {
-        ReportClaimMessagePayload payload = buildReportClaimMessagePayload(claim);
+    public void sendReportClaimMessage(Claim claim, List<LocalDate> datesOfBirthOfChildren, ClaimAction claimAction) {
```

Comment:
Would this method be less clunky if it was simply called `reportClaim`?

Reply:
It's not reporting the claim, just creating the message on the queue. Reporting the claim will be inside the MIReporter class.